src/OidcClient/AuthorizeClient.cs

```diff
@@ -86,7 +85,7 @@ public AuthorizeClient(OidcClientOptions options)
             return await _options.Browser.InvokeAsync(browserOptions, cancellationToken);
         }
 
-        public AuthorizeState CreateAuthorizeState(Parameters frontChannelParameters)
+        public async Task<AuthorizeState> CreateAuthorizeStateAsync(Parameters frontChannelParameters)
```

Comment:
We need this breaking change because we are now doing async work here (calling the par endpoint). I think this breaking change results in the best API, but it is a breaking change.

Options I see:
- Accept the breaking change, since this is a major release
- Keep the old method for backwards compatibility
    - Maybe also mark the old method as deprecated
    - And maybe check if par is enabled when calling the old method. If so, we can't respect the combination of options and the user is clearly already making changes and trying to use the new API, so we can throw an error

Reply:
I'd say make the breaking change, and drop the old method. It's so low level that the chances of people using it directly are low, and we don't document it formally anyhoo.

Reply:
Ping @leastprivilege for any further input.

Reply:
We discussed and all agree to make this change.